app/views/support_interface/application_forms/address_details/edit.html.erb

```diff
@@ -7,20 +7,42 @@
       <%= f.govuk_error_summary %>
 
       <%= f.govuk_fieldset legend: {text: t('support_interface.edit_address_details_form.address_details.label'), size: 'xl', tag: 'h1'} do %>
-        <% if @details_form.uk? %>
-          <%= f.govuk_text_field :address_line1, label: -> { safe_join([t('application_form.contact_information.address_line1.uk.label'), ' ', tag.span(t('application_form.contact_information.address_line1.hidden'), class: 'govuk-visually-hidden')]) }, autocomplete: 'address-line1' %>
-          <%= f.govuk_text_field :address_line2, label: {text: t('application_form.contact_information.address_line2.uk.label'), hidden: true}, autocomplete: 'address-line2' %>
-          <%= f.govuk_text_field :address_line3, label: {text: t('application_form.contact_information.address_line3.uk.label')}, width: 'two-thirds', autocomplete: 'address-level2' %>
-          <%= f.govuk_text_field :address_line4, label: {text: t('application_form.contact_information.address_line4.uk.label')}, width: 'two-thirds', autocomplete: 'address-level1' %>
-          <%= f.govuk_text_field :postcode, label: {text: t('application_form.contact_information.postcode.uk.label')}, width: 10, autocomplete: 'postal-code' %>
-        <% else %>
+
+        <% if @details_form.international? && @details_form.international_address %>
```

Comment:
I don't think it would hurt to always display this field for international addresses, that way it'll be a bit more obvious to support users whether a given candidate has a legacy `international_address` or not. Longer term we remove it altogether.

Reply:
Sure, I can see that might be useful. I'll make the other change as well then to keep the old address